src/views/core/members/UpdateProfile.vue

```diff
@@ -0,0 +1,217 @@
+              <li v-for="(error, key) in validationErrors" :key="key"><strong>{{ key }}</strong>: {{ error}}</li>
+            </ul>
+          </div>
+        </div>
+      </template>
+
+      <!-- <form @submit.prevent="saveUser()" v-if="validationErrors.email">
+        <select-or-custom v-model="user.gender" data-cy="gender" :values="['Male', 'Female']" label="Gender">
+          <template slot="errors-slot">
+            <p class="help is-danger" v-if="errors.gender">{{ errors.gender.join(', ')}}</p>
+          </template>
+        </select-or-custom>
+
+        <div class="field">
+          <label class="label">Address</label>
+          <div class="control">
+            <input class="input" data-cy="address" type="text" v-model="user.address" />
+          </div>
+          <p class="help is-danger" v-if="errors.address">{{ errors.address.join(', ')}}</p>
+        </div>
+
+        <div class="field">
+          <label class="label">About me</label>
+          <div class="control">
+            <input class="input" data-cy="about_me" type="text" v-model="user.about_me" />
+          </div>
+          <p class="help is-danger" v-if="errors.about_me">{{ errors.about_me.join(', ')}}</p>
+        </div>
+
+        <b-loading :is-full-page="false" :active.sync="isLoading"></b-loading>
+
+        <div class="field">
+          <div class="control">
+            <input type="submit" value="Save user" :disabled="isSaving" class="button is-primary is-fullwidth"/>
+          </div>
+        </div>
+      </form> -->
+
+      <hr v-if="validationErrors.email" />
+
+      <div class="field" v-if="validationErrors.email">
+        <label class="label">Email <span class="has-text-danger">*</span></label>
+        <div class="field has-addons">
+          <div class="control is-expanded">
+            <input class="input" type="email" v-model="tmpEmail" />
+          </div>
+          <div class="control">
+            <button class="button is-info" @click="changeEmail(tmpEmail)">Change email</button>
+          </div>
+        </div>
+      </div>
+
+      <hr v-if="validationErrors.bodies" />
+
+      <div class="field" v-if="validationErrors.bodies">
+        <label class="label">Body <span class="has-text-danger">*</span></label>
+        <div class="field has-addons">
+          <div class="control is-expanded">
+            <div class="select is-fullwidth">
+              <select v-model="tmpBodyId">
+                <option v-for="body in bodies" v-bind:key="body.id" :value="body.id">{{ body.name }}</option>
+              </select>
+            </div>
+          </div>
+          <div class="control">
+            <button class="button is-info" @click="askToJoinBody(tmpBodyId)">Join body</button>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import _ from 'lodash'
+import { mapGetters } from 'vuex'
+
+export default {
+  name: 'UpdateProfile',
+  data () {
+    return {
+      user: {
```

Comment:
Do we need to return all this data?

Reply:
we'll most likely need it later when editing user